core/include/traccc/edm/nseed.hpp

```diff
@@ -55,7 +58,7 @@ struct nseed {
      */
     const link_type* cend() const { return &_sps[_size]; }
 
-    private:
+    public:
     std::size_t _size;
     std::array<link_type, N> _sps;
```

Comment:
If client code is meant to access these variables now, they should really not be prefixed with `_`.

Reply:
Absolutely correct, I plan on renaming these and making some more changes to the `nseed` class during the development of this PR. Well spotted. :+1: